Storage/Rust/RustRemoteTabs.swift

```diff
@@ -173,4 +211,22 @@ public extension ClientRemoteTabs {
     func toClientAndTabs(client: RemoteClient) -> ClientAndTabs {
         return ClientAndTabs(client: client, tabs: self.remoteTabs.map { $0.toRemoteTab(client: client)}.compactMap { $0 })
     }
+
+    func toClientAndTabs() -> ClientAndTabs {
+        let client = self.toRemoteClient()
+        let tabs = self.remoteTabs.map { $0.toRemoteTab(client: client)}.compactMap { $0 }
+
+        return ClientAndTabs(client: client, tabs: tabs)
```

Comment:
nit: sometimes compiler doesn't catch on return so adding an object before that would give better debug output

```suggestion
        let clientAndTabs = ClientAndTabs(client: client, tabs: tabs)
        return clientAndTabs
```